templates/product/index.html.twig

```diff
@@ -0,0 +1,65 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}{{ 'product.title'|trans }}{% endblock %}
+
+{% import "_partials.html.twig" as macros %}
+
+{% block content %}
+    <h1 class="page-title">{{ 'product.title'|trans }}</h1>
+
+    {{ form_start(form) }}
+    <div class="form-default">
+        {{ form_row(form.name) }}
+        {{ form_row(form.project) }}
+        <div>
+            <button class="button form-element">{{ 'client.search'|trans }}</button>
+        </div>
+    </div>
+    {{ form_end(form) }}
+
+    {% set number_of_columns = 7 %}
+
+    <table class="table margin-bottom">
+        <thead class="table-th">
+        <tr>
+            <th class="table-th{% if products.isSorted('product.id') %} sorted{% endif %}">{{ knp_pagination_sortable(products, 'product.id'|trans, 'product.id') }}</th>
```

Comment:
Shouldn't these (and all the other sortable th) use the macro knp_pagination_sortable_header ?